### PR TITLE
Generate random weave password it none is supplied

### PR DESCRIPTION
**Pull request by kampka**
Addresses the enhancements proposed by @justinsb in #5441

### CODE REVIEW

File: docs/networking.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ spec:
 
 The Weave network encryption is configurable by creating a weave network secret password.
 Weaveworks recommends choosing a secret with [at least 50 bits of entropy](https://www.weave.works/docs/net/latest/tasks/manage/security-untrusted-networks/).
+I no password is supplied, kops will generate one at random.
```
Comment (rdrgmnzs) [inline docs/networking.md]: Typo, should be `If`.

**Comment by rdrgmnzs** [comment]
Looks like a run of `make gen-cli-docs` is needed.

**Comment by kampka** [comment]
@rdrgmnzs Indeed, sorry about that.
Should be fixed now.

**Comment by rdrgmnzs** [comment]
@kampka thanks for the work! 

/lgtm

**Comment by justinsb** [comment]
Thanks @kampka - awesome!

/approve
/lgtm

**Pull request merged**